src/main/flight/mixer.c

```diff
@@ -661,6 +661,130 @@ void StopPwmAllMotors()
 }
 
 #ifndef USE_QUAD_MIXER_ONLY
+static int16_t decidegreeToServoPulse(uint8_t servo_index, int16_t angle_deci_degree)
+{
+    // normalize angle to range -180 to 180 (degree)
+    while (angle_deci_degree > 1800)
+        angle_deci_degree -= 3600;
+    while (angle_deci_degree < -1800)
+        angle_deci_degree += 3600;
+
+    //remap input value (RX limit) to output value (Servo limit), also take into account eventual non-linearity of the two half range
+    if (angle_deci_degree > 0) {
+        angle_deci_degree = scaleRange(angle_deci_degree, 0, servoConf[servo_index].angleAtMax * 10, 0, servoConf[servo_index].max);
+    } else {
+        angle_deci_degree = scaleRange(angle_deci_degree, 0, servoConf[servo_index].angleAtMin * 10, 0, servoConf[servo_index].min);
```

Comment:
The subtraction will be still necessary before summing values. And current code assumes different scaling for each half of range ...
But this whole code could use big rewrite ...